jupyterlab_git/git.py

```diff
@@ -955,8 +955,13 @@ async def show_prefix(self, path):
             cmd,
             cwd=path,
         )
+
         if code == 0:
-            result = {"code": code, "path": my_output.strip("\n")}
+            result = {
+                "code": code,
+                "path": my_output.strip("\n"),
+                "relative_path": os.path.relpath(path, contents_manager.root_dir),
+            }
```

Comment:
Thanks @Zxun2 

This is going in the right direction. However I would encapsulate all the logic here on the server side.

So the code will be something like:
```suggestion
            relative_path = os.path.relpath(path, contents_manager.root_dir)
            result = {
                "code": code,
                "path": None if relative_path == "." else my_output.strip("\n"),
            }
```